### DISCUSSION TITLE
How to get a Free Domain Name?

### DISCUSSION THREAD

**Discussion by DOodle25** [community/community#122521] [Other Feature Feedback, Questions, & Ideas]
# Free DOMAIN NAME

🌐 **Hello Fellow GitHub Devs!** 🌐

Today, let's dive into the realm of domain names! We all know that having a catchy domain name can make a world of difference for your project's visibility. But here's the twist: **do you know how to snag one for free?** 🆓💻

💡 Whether you're a seasoned developer or just starting out, there are numerous ways to get your hands on a free domain name. From GitHub Pages to various domain registrars offering promotions, the options are aplenty! 💡

🤔 So, dear community, let's pool our knowledge! **How many ways do you know to acquire a free domain name for your GitHub projects?** Share your insights, tips, and tricks below! 📝✨

Personally, I've explored a few avenues, and I'm even considering creating a GitHub community or repository to compile these methods. If you're interested in contributing or learning more, please connect with me! Let's collaborate and make our projects even better together. 

**Please follow me so I can follow you back and connect with you more!**

Keep coding! 💻
- DOodle25

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @DOodle25 !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the `General` category. I've gone ahead and moved it for you. Good luck!

**Comment by MERT-CKR** [comment]
I'm not sure about the domain aspect, but you can get free hosting from here: https://www.freehosting.com/

**Reply by DOodle25** [reply]
That amazing.... though does this only allow with a domain name also

**Comment by SamBeasley1026** [comment]
MetaMask wallet. 


You can get free url dns 


Which is what Freename.io uses

**Reply by DOodle25** [reply]
I went through it but i didn't find any way to get a free domain name

**Comment by SamBeasley1026** [comment]
I had 58,000,000 blockchains on one of nine of my URLs 


126,000 deposits 350,000 some odd deposits blockchained last seventy min

**Comment by DOodle25** [comment]
Though i did found 2 ways to get free domain name from one of which i claimed 1 for free for 1 year and another which provides you domain name for less price for the first time you purchase it

**Comment by DOodle25** [comment]
certain website like porkbun there is coupon code which after participating and monthly happening hackathon you can get a code to redeem a free domain name on their site only 1 time after your verification
